app/crud/reviews.py

```diff
@@ -65,23 +37,36 @@ async def create_review_in_db(session: AsyncSession, owner_id: int, review_data:
     result = await session.execute(
         insert(review_table)
         .values(owner_id=owner_id, last_edit_date=datetime.date.today(), **review_data.model_dump())
-        .returning(review_table.c[*Review.model_fields])
+        .returning(*[review_table.c[field] for field in REVIEW_FIELDS])
     )
+    current_avg = book['avg_mark']
+    reviews_count_for_book = book['marks_count']
+    new_reviews_count = reviews_count_for_book + 1
+    new_avg = (current_avg * reviews_count_for_book + review_data.mark) / new_reviews_count
+    await update_book_in_db(session, review_data.book_id,
+                            BookUpdate(**{'avg_mark': new_avg, 'marks_count': new_reviews_count}))
```

Comment:
```suggestion
                            BookUpdate(avg_mark=new_avg, marks_count=new_reviews_count))
```

![539e85af2ff51f073f9e8bc9a2d85c50](https://github.com/user-attachments/assets/8b40ae55-37cf-479c-b144-76b04d94f55e)